site/content/markdown/feedback.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Digital Collections Feedback
+slug: feedback
+menu: feedback
+layout: markdownWithMenu
```

Comment:
I'm guessing this is a test file that can be removed, but `layout` is no longer supported in the markdown rendering since we refactored.